docs/source/libraries.rst

```diff
@@ -9,53 +9,53 @@ Supported Peripherals
-| TWAI          | No            | No            | No            |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| UART          | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| USB           | Not Supported | Yes           | Yes           | ESP32-C3 only CDC/JTAG        |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Wi-Fi         | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | ESP32-S3      | Comments               |
++===============+===============+===============+===============+===============+========================+
+| ADC           | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Bluetooth     | Yes           | Not Supported | Not Supported | Not Supported | Bluetooth Classic      |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| BLE           | Yes           | Not Supported | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| DAC           | Yes           | Yes           | Not Supported | Not Supported |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Ethernet      | Yes           | Not Supported | Not Supported | Not Supported | (*)                    |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| GPIO          | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Hall Sensor   | Yes           | Not Supported | Not Supported | Not Supported |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| I2C           | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| I2S           | Yes           | No            | No            | Yes           | WIP                    |
```

Comment:
I think that ESP32-S2 has I2S support... please check with @PilnyTomas

Reply:
Fixed :) All chips has I2S supported.

Reply:
I saw that the I2S example excludes the C3 with the .skip.esp32c3 tag file. Maybe Pilny's library doesn't support this SoC.

Reply:
I got info from him that it supports :) but C3 has only 1 I2S. Maybe the example is made using both I2S peripherals :)